### PR TITLE
Logstash integration

### PR DESCRIPTION
**Pull request by jamespamplin**
Basic logstash integration. Ensures all logs are persisted to our Logstash service.

### CODE REVIEW

File: app/com/gu/viewer/config/Configuration.scala [hunk 1]
```diff
@@ -13,13 +13,20 @@ object Configuration {
       sys.error(s"Config key required: $key")
     }
 
+  lazy val app: String = AWS.readTag("App").getOrElse("viewer")
+
   lazy val stage: String = AWS.readTag("Stage") match {
```
Comment (sihil) [inline app/com/gu/viewer/config/Configuration.scala]: getOrElse like the others?

File: conf/logback.xml [hunk 2]
```diff
@@ -15,7 +15,7 @@
   <logger name="play" level="INFO" />
   <logger name="application" level="DEBUG" />
 
-  <root level="ERROR">
+  <root level="INFO">
     <appender-ref ref="STDOUT" />
```
Comment (sihil) [inline conf/logback.xml:19]: Don't log info to stdout - makes it really hard to rotate files (and if we don't then we could fill up the disk and stop working). Set up a file logger (see flex for an example) and attach the root logger to that.

Comment (sihil) [inline conf/logback.xml:19]: (I've noticed us logging to stdout in many places and think we need to deprecate the practice.)

**Comment by sihil** [comment]
Did you consider using the kinesis stream for the purposes of magic?

File: app/com/gu/viewer/config/Configuration.scala (hunk 1 above)
Comment (jamespamplin) [inline app/com/gu/viewer/config/Configuration.scala]: Done.

File: conf/logback.xml (hunk 2 above)
Comment (jamespamplin) [inline conf/logback.xml:19]: Thanks, good advice. Forgot the console logs appended to a file automatically with play. For now I've reset this back to `ERROR`. Can add a rolling file appender later.

**Comment by jamespamplin** [comment]
@sihil thought direct tcp ingest would be easier to setup. Unless theres a massive benefit, I'd rather not use kinesis as Viewer has no other aws components apart from basic EC2 and ELB so far. No need to overcomplicate things.

**Comment by jamespamplin** [comment]
Tested locally, looks ok

**Pull request merged**